SocialCareCaseViewerApi.Tests/V1/Controllers/RelationshipControllerTests.cs

```diff
@@ -0,0 +1,74 @@
+using AutoFixture;
+using Bogus;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.V1.Boundary.Requests;
+using SocialCareCaseViewerApi.V1.Boundary.Response;
+using SocialCareCaseViewerApi.V1.Controllers;
+using SocialCareCaseViewerApi.V1.Exceptions;
+using SocialCareCaseViewerApi.V1.UseCase.Interfaces;
+
+namespace SocialCareCaseViewerApi.Tests.V1.Controllers
+{
+    [TestFixture]
+    public class RelationshipControllerTests
+    {
+        private RelationshipController _classUnderTest;
+        private Mock<IRelationshipsV1UseCase> _mockRelationshipsUseCase;
+        private Fixture _fixture;
+        private Faker _faker;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockRelationshipsUseCase = new Mock<IRelationshipsV1UseCase>();
+
+            _classUnderTest = new RelationshipController(_mockRelationshipsUseCase.Object);
+
+            _fixture = new Fixture();
```

Comment:
You could move the `new Fixture()` to where the field is declared and it can then be made `readonly`, [SetUp] is ran before every test so before each test we make a new Fixture object and assign it to _fixture.

`private readonly Fixture _ fixture = new Fixture()`

^ same should work for _faker.

Reply:
Just tested, all good! This is nicer as it makes the `SetUp` method leaner. 👍